src/posts/introducing-inference-data/index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: 'Better Bayesian Workflows with InferenceData and Xarray'
+date: '2022-12-17'
```

Comment:
```suggestion
date: '2023-03-06'
```

Reply:
Yes! Love the enthusiasm

Reply:
Pick a new day and we'll work to make it happen!

Reply:
Done 
```suggestion
date: '2023-03-14'
```

Reply:
Done 
```suggestion
date: '2023-03-21'
```

@OriolAbril think we can get this done in the next two days?